### PR TITLE
Update Chess Presence

### PR DESCRIPTION
**Pull request by Funeoz**
Here is an update for the presence to handle new pages on Chess.com. 

image
image
image
image

### CODE REVIEW

File: websites/C/Chess/presence.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 var presence = new Presence({
-    clientId: "699204548664885279"
-  }),
+  clientId: "699204548664885279"
+}),
```
Comment (Bas950) [inline websites/C/Chess/presence.ts:3]: no var, use const or let

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/C/Chess/presence.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-var presence = new Presence({
+let presence = new Presence({
```
Comment (Bas950) [inline websites/C/Chess/presence.ts]: const

Comment (Bas950) [reply websites/C/Chess/presence.ts]: fix it at other places in the code too. No var :/

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/C/Chess/presence.ts (hunk 2 above)
Comment (Funeoz) [reply websites/C/Chess/presence.ts]: Oops. Ok I will fix that.

**Comment by Funeoz** [comment]
I don't really know how to replace `any` for this latest check...

File: websites/C/Chess/presence.ts [hunk 3]
```diff
@@ -15,15 +15,15 @@ function getTimestamps(
   videoTime: number,
   videoDuration: number
 ): Array<number> {
-  var startTime = Date.now();
-  var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  const startTime = Date.now();
+  const endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
```
Comment (Bas950) [inline websites/C/Chess/presence.ts:19]: merge consts
const var1 = obj,
var2 = obj;

File: websites/C/Chess/presence.ts [hunk 4]
```diff
@@ -132,15 +142,22 @@ presence.on("UpdateData", async () => {
...
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Watching Computer Chess Championship";
+    presenceData.state = document.title.substring(0, document.title.indexOf("-"));
   } else if (document.location.pathname.indexOf("/video/player/") == 0) {
-    var video: HTMLVideoElement = document.querySelector("video");
+    const video: HTMLVideoElement = document.querySelector("video");
 
     if (video !== null && !isNaN(video.duration)) {
-      var timestamps: any;
+      let timestamps: any;
```
Comment (Bas950) [inline websites/C/Chess/presence.ts:160]: array<number>

**Review by Bas950 — APPROVED** [review]

**Review by Alanexei — APPROVED** [review]

**Pull request merged**